submissions/natashafir/friends-app/index.js

```diff
@@ -0,0 +1,107 @@
+const URL = 'https://randomuser.me/api/?results=50';
+const CARDS = document.querySelector('.cards');
+let filterBy = 'all';
+let allFriends = [];
+let friends = [];
+let sortedFriends;
+const input = document.querySelector('input');
+
+function initialApp() {
+    fetch(URL)
+        .then(response => response.json())
+        .then((data) => (allFriends = data.results))
+        .then(() => userTemplate(allFriends))
+        .then(()=> renderUsers(allFriends))
```

Comment:
You should handle fetch response status - https://www.tjvantoll.com/2015/09/13/fetch-and-errors/